requirements.txt

```diff
@@ -1,4 +1,8 @@
-GSI>=0.6.4
+# From repo
+git+https://:@gitlab.cern.ch:8443/fts/fts-rest.git#egg=fts3
```

Comment:
No, I'd rather not. This file is only used for development and for running the unit tests. Indirectly, it'd end up being used for debugging new versions of FTS before making it to a release. So, I prefer leaving the "last" version.